lib/routes/account.js

```diff
@@ -1131,11 +1136,24 @@ module.exports = function (
                * 3) Verify account email if not already verified.
                */
               return db.verifyTokens(code, account)
+                .then(function () {
+                  return log.activityEvent('account.signin.verify.success', request, {
+                    uid: account.uid.toString('hex'),
+                    code: request.payload.code
+                  })
+                })
                 .catch(function (err) {
                   if (err.errno === error.ERRNO.INVALID_VERIFICATION_CODE && butil.buffersAreEqual(code, account.emailCode)) {
                     // The code is just for the account, not for any sessions
                     return true
                   }
+
+                  log.activityEvent('account.signin.verify.error', request, {
```

Comment:
I'm not sure about using `activityEvent` for these, since that means it will wind up in our s3->redash pipeline.  We probably want it there someday, but doing a simple `log.info({ op: 'account.signin.verify.success', ...})` and `log.error({ op: 'acount.signin.verify.error', ... })` is probably simpler for now (and will mean we can start getting the results straight away, rather than waiting for the weekly dump to S3).